test/nft/ZoraCreator1155.t.sol

```diff
@@ -156,7 +150,7 @@ contract ZoraCreator1155Test is Test {
     function test_contractVersion() external {
         init();
 
-        assertEq(target.contractVersion(), "1.4.0");
+        assertEq(target.contractVersion(), "1.4.1");
```

Comment:
how about doing the version bump in a separate pr, when we decide to publish/release the new version?

Reply:
yep reverted